src/contacts/EditContact/EditContactContainer.js

```diff
@@ -1,48 +1,33 @@
 import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
+
 import { get } from 'lodash';
 
 import {
-  Button,
-  Pane,
-  PaneMenu,
-  Row,
-  Col,
-} from '@folio/stripes/components';
-import { AppIcon } from '@folio/stripes/core';
-
-import { categoriesResource, contactResource } from '../../common/resources';
+  categoriesResource,
+  contactResource,
+} from '../../common/resources';
+import { saveContact } from '../ViewContact/util';
 import EditContact from './EditContact';
 
-function getLastMenu(handleSubmit) {
-  return (
-    <PaneMenu>
-      <FormattedMessage id="ui-organizations.contacts.button.submit">
-        {(title) => (
-          <Button
-            buttonStyle="primary"
-            marginBottom0
-            onClick={handleSubmit}
-            title={title}
-            type="submit"
-          >
-            {title}
-          </Button>
-        )}
-      </FormattedMessage>
-    </PaneMenu>
-  );
-}
-
 class EditContactContainer extends Component {
   static manifest = Object.freeze({
     contact: contactResource,
     categories: categoriesResource,
   });
 
+  onSubmit = (contact) => {
+    const { mutator, onClose, showMessage } = this.props;
+
+    saveContact(mutator.contact, contact)
+      .then(() => showMessage('ui-organizations.contacts.message.saved.success', 'success'))
```

Comment:
I'm thinking if displaying this message makes sense at all. Is user able to see it in case the next `then` and transition to previous page?

Reply:
The message is shown on the view contact screen